src/components/LayoutHero/LayoutHero.astro

```diff
@@ -0,0 +1,45 @@
+---
+import { Button, buttonVariants, Typography } from "@chainlink/blocks"
+import styles from "./LayoutHero.module.css"
+
+interface Props {
+  title: string
+  description: string
+  buttons: Array<{
+    label: string
+    link: string
+  }>
+  image: string
+}
+
+const { title, description, buttons = [], image } = Astro.props as Props
+---
+
+<section class={styles.layoutHero}>
+  <div class={styles.heroContentWrapper}>
+    <div class={styles.heroContent}>
+      <h2 class={styles.heroTitle}>{title}</h2>
+      <Typography variant="body-s" color="muted">{description}</Typography>
+      <div class={styles.heroButtons}>
+        {
+          buttons.map((button, index) => (
+            <a
+              href={button.link}
+              class={buttonVariants({
+                variant: index === 0 ? "primary" : "tertiary",
+                size: "sm",
+              })}
+            >
+              {button.label}
+            </a>
```

Comment:
Should not be using a block button?

Reply:
The way that they look to be used in the UI is as links. Being semantically correct, should we not use anchors with button styling (for navigating) rather than buttons? I think doing <Button asChild/> would be inconsequential compared to doing it this way. let me know what you think

Reply:
Ok! That results in less code too so I will do it that way